plugin/storage/es/mappings/jaeger-span.json

```diff
@@ -64,7 +64,9 @@
         "flags":{
           "type":"integer"
         },
-        "logs":{
+        "logs" : {
```

Comment:
Although I am fine to add the `type: "nested"` as it is really a nested type.

https://www.elastic.co/guide/en/elasticsearch/reference/current/nested.html

> The nested type is a specialised version of the object datatype that allows arrays of objects to be indexed in a way that they can be queried independently of each other.

```
"logs" : [
            {
              "timestamp" : 1561112938308666,
              "fields" : [
                {
                  "key" : "event",
                  "type" : "string",
                  "value" : "GetConn"
                }
              ]
            },
            {
              "timestamp" : 1561112938308674,
              "fields" : [
                {
                  "key" : "event",
                  "type" : "string",
                  "value" : "GotConn"
                }
              ]
            },
            {
              "timestamp" : 1561112938308721,
              "fields" : [
                {
                  "key" : "event",
                  "type" : "string",
                  "value" : "WroteHeaders"
                }
              ]
            },
            {
              "timestamp" : 1561112938308724,
              "fields" : [
                {
                  "key" : "event",
                  "type" : "string",
                  "value" : "WroteRequest"
                }
              ]
            },
            {
              "timestamp" : 1561112938377614,
              "fields" : [
                {
                  "key" : "event",
                  "type" : "string",
                  "value" : "GotFirstResponseByte"
                }
              ]
            },
            {
              "timestamp" : 1561112938377752,
              "fields" : [
                {
                  "key" : "event",
                  "type" : "string",
                  "value" : "PutIdleConn"
                }
              ]
            },
            {
              "timestamp" : 1561112938377812,
              "fields" : [
                {
                  "key" : "event",
                  "type" : "string",
                  "value" : "ClosedBody"
                }
              ]
            }
          ],
```